Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/GameObject.hpp

```diff
@@ -23,8 +22,8 @@ namespace sne
 	class GameObject
 	{
 	private:
-		Vector3 position; //TODO: remove 
-		Vector3 rotation; //TODO: remove - Radian
+		glm::vec3 position; 
+		glm::vec3 rotation; // Radian
```

Comment:
For now, the position is also stored in the "model" matrix. You will need to change the translate method to move the position vector and then translate the matrix.
And do the same for the rotation matrix

Reply:
Ok, so if i'm right: 
mat 4x4 => 3*3 for rotation 3*1 for translation and the last line because: because ? :joy:

Reply:
I'll make the change. Could you just confirm me before that:
Your objects and points have relative coordinates and this matrix is usefull 'only' to evalute/positionne them in the global game ?

Reply:
object position are their position in the world (assuming we don't change the world center)

For 4*4 matrix the last line is usefull only to be able to mulitply matrices.